modules/git/commit_submodule_file.go

```diff
@@ -101,8 +103,8 @@ func getRefURL(refURL, urlPrefix, repoFullName, sshDomain string) string {
 
 // RefURL guesses and returns reference URL.
 // FIXME: template passes AppURL as urlPrefix, it needs to figure out the correct approach (no hard-coded AppURL anymore)
-func (sf *CommitSubModuleFile) RefURL(urlPrefix, repoFullName, sshDomain string) string {
-	return getRefURL(sf.refURL, urlPrefix, repoFullName, sshDomain)
+func (sf *CommitSubModuleFile) RefURL(repoFullName string) string {
+	return getRefURL(sf.refURL, setting.AppURL, repoFullName, setting.SSH.Domain)
```

Comment:
Thank you very much for improving it (IIRC I left that "FIXME" 🤣 )

Actually the `getRefURL` should also be rewritten, will do it later in this PR together.